source/view.js

```diff
@@ -4,39 +4,82 @@
 
 /********************** View **********************/
 "use strict";
+
 function View(){
-    let cells = document.getElementsByClassName("cell");
-    let statusField = document.getElementById("gameStatus");
     let viewModel = new ViewModel();
-    let occupiedCells = [];
+    let cells = document.getElementsByClassName("cell");
+    let occupiedCells;
+
+    function getFieldCellsArr(){
+        return viewModel.gameInfo.getFieldCells();
+    };
 
     this.init = function(){
-        viewModel.init();
+        let rowsUserSet = viewModel.rowsInField;
+        let columnsUserSet = viewModel.columnsInField;
+        viewModel.init(rowsUserSet, columnsUserSet);
+        createPlayField(rowsUserSet, columnsUserSet);
+        createOccupiedCells(rowsUserSet, columnsUserSet);
         setListenersToCells();
         changeCurrentStatus();
     };
 
+    function createPlayField(rows, columns){
+        let rowsCount = rows;
+        let columnsCount = columns;
+        let playField = document.getElementById('play_field');
+        let fieldTbl = document.createElement('table');
+
+        playField.appendChild(fieldTbl);
+
+        for (let r = 0; r < rowsCount; r++) {
+            let rowField = document.createElement('tr');
+            rowField.setAttribute('class', 'row');
+            fieldTbl.appendChild(rowField);
+            for (let c = 0; c < columnsCount; c++) {
+                let cellField = document.createElement('td');
+                cellField.setAttribute('class', 'cell');
+                cellField.setAttribute('id', r+'-'+c);
+                rowField.appendChild(cellField);
+            }
+        }
+    };
+
+    function createOccupiedCells(rows, columns) {
+        let arr = new Array(rows);
+        for (let i = 0; i < rows; i++){
+            arr[i] = new Array(columns);
+        }
+        occupiedCells = arr;
+    };
+
     function setListenersToCells(){
         for (let i = 0; i < cells.length; i++) {
             cells[i].addEventListener("click", function(){
-                viewModel.finishTurn(i);
+                let cellsRow = cells[i].id.split('-')[0];
+                let cellsCol= cells[i].id.split('-')[1];
+                viewModel.finishTurn(+cellsRow, +cellsCol);
                 updateCellContent();
                 changeCurrentStatus();
             });
         }
     };
 
-    function changeCurrentStatus(){
-        statusField.innerHTML = viewModel.gameInfo.getCurrentStatus();
-    };
-
     function updateCellContent(){
-        let fieldCellsArr = viewModel.gameInfo.getFieldCells();
-        for (let i = 0; i < fieldCellsArr.length; i++) {
-            if (fieldCellsArr[i] != undefined && occupiedCells[i] != fieldCellsArr[i]) {
-                cells[i].innerHTML = fieldCellsArr[i];
+        for (let r = 0; r < getFieldCellsArr().length; r++) {
```

Comment:
`getFieldCellsArr` method can be executed once and its value can be stored in a variable. Later this variable can be used instead of the `getFieldCellsArr` method. It will minimize CPU load.